cmd/tealdbg/debugger.go

```diff
@@ -58,9 +59,10 @@ type Control interface {
 	RemoveBreakpoint(line int) error
 	SetBreakpointsActive(active bool)
 
-	GetSourceMap() ([]byte, error)
 	GetSource() (string, []byte)
 	GetStates(s *logic.DebugState) AppState
+
+	logic.SourceMapper
```

Comment:
I like having Mapper interface. But not sure if mixing in a Mapper interface into a Control interface a good idea since looks like no benefits compared to older approach with the `GetSourceMap` member